k8s/base/juno/juno-db-snapshot.dockerfile

```
@@ -0,0 +1,10 @@
+FROM alpine:3.20
+
+RUN apk update && apk add --no-cache wget tar
+
+WORKDIR /snapshots
+
+RUN wget -O juno_sepolia.tar https://juno-snapshots.nethermind.dev/sepolia/juno_sepolia_v0.11.7_66477.tar && \
```

Hadolint

[info] Avoid use of wget without progress bar. Use `wget --progress=dot:giga <url>`. Or consider using `-q` or `-nv` (shorthands for `--quiet` or `--no-verbose`).

(DL3047)